refactron/core/refactron.py

```
@@ -469,6 +522,52 @@ def _analyze_file(self, file_path: Path) -> FileMetrics:
+                    if matched_confidence >= 0.5:
+                        filtered_issues.append(issue)
+                    else:
+                        logger.debug(
+                            f"LLM suppressed false positive: {issue.message} (score: {matched_confidence})"
+                        )
+
+                metrics.issues = filtered_issues
+                logger.debug(
+                    f"Completed LLM triage in {time.time() - start_triage:.2f}s. Kept {len(metrics.issues)} issues."
+                )
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)